src/Game.java

```diff
@@ -1,45 +1,119 @@
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
 import java.util.Scanner;
+import java.util.function.Function;
+import java.util.stream.Collectors;
 
 public class Game {
-    public static void main(String[] args) {
-        System.out.println("Welcome to PPTHangman!");
-        System.out.println("Please enter the number of players:");
 
-        Scanner scanner = new Scanner(System.in);
+  public static Map<String, Integer> leaderboardToSet(List<Player> allPlayers) {
+    return allPlayers.stream()
+        .collect(Collectors.toMap(Player::toString, Player::getScore));
+  }
 
-        int numPlayers = Integer.parseInt(scanner.nextLine());
-        List<Player> allPlayers = new ArrayList<>();
+  public static void main(String[] args) {
+    System.out.println("Welcome to PPTHangman!");
+    System.out.println("Please enter the number of players:");
 
-        for (int i = 0; i < numPlayers; i++) {
-            System.out.println("Please enter the name of the " + (i + 1) + "th player");
-            allPlayers.add(new Player(scanner.nextLine()));
+    Scanner scanner;
+    int numPlayers;
+    String numPlayersString;
+
+    while (true) {
+      scanner = new Scanner(System.in);
+      numPlayersString = scanner.nextLine();
+
+      if (numPlayersString.length() == 0){
+        System.out.println("Invalid input, please insert a number:");
+      } else {
+        numPlayers = Integer.parseInt(numPlayersString);
+        if (numPlayers > 1) {
+          break;
+        } else if (numPlayers == 1) {
+          System.out.println(
+              "You cannot play alone! Please enter a number greater than 1:");
```

Comment:
Do you really need this special case, just to say "you cannot play alone?" Could me merged with the case below.